google-cloud-core-bom/pom.xml

```diff
@@ -75,6 +75,11 @@
         <artifactId>google-cloud-core-http</artifactId>
         <version>2.3.2-SNAPSHOT</version><!-- {x-version-update:google-cloud-core:current} -->
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-core-native-support</artifactId>
```

Comment:
Would you clarify the reasons for this artifact renaming?

Reply:
Hmm good point. I originally wanted to fit into naming conventions in this repo. But I think there will be convenience benefits for keeping artifact IDs the same. So I renamed it back to `native-image-support`.